### PR TITLE
automatic harvest

### PR DESCRIPTION
**Pull request by rerowep**
For the automatic harvest CELERY and BEAT must be started:

`./scripts/worker_beat`

### CODE REVIEW

File: rero_ebooks/tasks.py [hunk 1]
```diff
@@ -31,19 +29,3 @@ def create_records(records, verbose=False):
         if verbose:
             click.echo('record uuid: ' + str(rec.id) + ' | ' + status)
```
Comment (jma) [inline rero_ebooks/tasks.py]: use `current_app.logger.info` instead.

File: scripts/setup [hunk 2]
```diff
@@ -29,3 +29,8 @@ invenio oaiharvester initconfig data/oaisources.yml
 dojson -l marcxml -i data/ebooks.xml do cantookmarc21 | invenio records create_or_update --vendor cantook
 # invenio index reindex -t ebook --yes-i-know
 # invenio index run
+
```
Comment (jma) [inline scripts/setup]: to be removed, but where is `invenio oaiharvester initconfig`?

Comment (rerowep) [reply scripts/setup]: Was already there:
`invenio oaiharvester initconfig data/oaisources.yml`

**Comment by jma** [comment]
Add scheduler to kubernetes and docker-compose.

**Pull request merged**